File: Amazon_LeetCode_Python_C++/ArrayString/1119_remove_vowels_from_a_string.cpp
class Solution {
public:
    string removeVowels(string s) {
        string t;
        for (int i = 0; i < s.size(); i++) {
            if (tolower(s[i]) != 'a' && tolower(s[i]) != 'e' && tolower(s[i]) != 'i' &&
                tolower(s[i]) != 'o' && tolower(s[i]) != 'u') {
                t += s[i];
            }
        }
        return t;
    }
};

File: Amazon_LeetCode_Python_C++/ArrayString/1119_remove_vowels_from_a_string.py
class Solution:
    def removeVowels(self, s: str) -> str:
        t = list()
        v = ['a', 'e', 'i', 'o', 'u']
        for i, c in enumerate(s):
            if c.lower() not in v:
                t.append(c)
                
        return ''.join(t)
